packages/gatsby-plugin-sass/src/__tests__/__snapshots__/gatsby-node.js.snap

```diff
@@ -12,29 +12,20 @@ exports[`gatsby-plugin-sass Stage: build-html / No options 1`] = `
                 Object {
                   "test": /\\\\\\.module\\\\\\.s\\(a\\|c\\)ss\\$/,
                   "use": Array [
-                    "miniCssExtract",
                     "css({\\"modules\\":true,\\"importLoaders\\":2})",
                     "postcss({})",
                     Object {
                       "loader": "/resolved/path/sass-loader",
                       "options": Object {
-                        "sourceMap": true,
+                        "sourceMap": false,
```

Comment:
Should this be false?

Reply:
this comes from gatsby core - where css sourceMaps are being produced only for `develop` (https://github.com/gatsbyjs/gatsby/blob/3dbec2c11dfd98c5a949a40ee193195785d7d3f3/packages/gatsby/src/utils/webpack-utils.js#L195), so if this should be changed it should be in separate PR